doc/internal/man3/bn_mul_words.pod

```diff
@@ -8,8 +8,9 @@ bn_sqr_comba4, bn_sqr_comba8, bn_cmp_words, bn_mul_normal,
 bn_mul_low_normal, bn_mul_recursive, bn_mul_part_recursive,
 bn_mul_low_recursive, bn_sqr_normal, bn_sqr_recursive,
 bn_expand, bn_wexpand, bn_expand2, bn_fix_top, bn_check_top,
-bn_print, bn_dump, bn_set_max, bn_set_high, bn_set_low - BIGNUM
-library internal functions
+bn_print, bn_dump, bn_set_max, bn_set_high, bn_set_low,
+mul, mul_add, sqr
```

Comment:
These seem to lack some prefixes and suffixes.

Reply:
they're macro's.  Added comments in the fixup